app/client/src/features/events/Questions/QuestionContent.tsx

```diff
@@ -18,7 +18,7 @@ export function QuestionContent({ fragmentRef, typographyProps = {}, ...props }:
     const questionContentData = useFragment(QUESTION_CONTENT_FRAGMENT, fragmentRef);
     return (
         <CardContent {...props}>
-            <Typography {...typographyProps}>{questionContentData.question}</Typography>
+            <Typography style={{ wordBreak: 'break-word' }} {...typographyProps}>{questionContentData.question}</Typography>
```

Comment:
Just an fyi, it's good to check css properties on here https://caniuse.com/mdn-css_properties_word-break_break-word. Also, this doesn't seem login/logout related?

Reply:
I probably should have just named the branch bugfix since it's general bug fixes but the login/logout was the primary one I was fixing there but I might move to another branch. And thanks that a good resource, I'm glad break-word is supported by most browsers because it fixes the cards going too wide.